lib/socket/socket.nit

```diff
@@ -334,26 +336,33 @@ end
 class SocketObserver
 	private var native = new NativeSocketObserver
 
+	# Set of FDs on which to watch read events
```

Comment:
`FD` does not seem to be defined anywhere, maybe `file descriptor` isn't that obvious?